src/decorators/blissComponent.js

```diff
@@ -41,8 +38,21 @@ export default ComposedComponent => class extends React.Component {
 		return classNames(classes);
 	}
 
+	normalizeModifiers(modifiers) {
+		if (!modifiers) return [];
+		return _.isArray(modifiers) ? modifiers : modifiers.split(/\s/i);
```

Comment:
like this, there will be empty modifier when multiple spaces are contained in value of `modifiers`. Make it  `/\s+/i` pls